playbooks/robusta_playbooks/job_actions.py

```diff
@@ -164,10 +172,13 @@ def job_pod_enricher(event: JobEvent, params: JobPodEnricherParams):
     pod = get_job_latest_pod(job)
 
     if not pod:
+        assert job.metadata is not None
```

Comment:
what about 
`        
        assert job.metadata.name is not None
        assert job.metadata.namespace is not None
`?